fetch/schema_creation.go

```diff
@@ -47,6 +47,12 @@ func (cs columnsWithType) CRDBCreateTableStmt(
 	includePkForEachCol := len(pkList) <= 1
 
 	for _, col := range cs {
+		if table.LocalityExpr == multiRegionRBRLocality && col.columnName == multiRegionHiddenColName {
+			// We don't explicitly create `crdb_region` table for RBR tables but let it be created
+			// automatically via the `LOCALITY REGIONAL BY ROW` clause.
+			logger.Info().Msgf("skipping creating hidden default column %s for table %s has locality %s", multiRegionHiddenColName, table.String(), table.LocalityExpr)
```

Comment:
```suggestion
			logger.Info().Msgf("skipping creation of the hidden default column %s for table %s with locality %s", multiRegionHiddenColName, table.String(), table.LocalityExpr)
```
I think that makes the log a bit cleaner.